resources/prosody-plugins/util.lib.lua

```diff
@@ -112,6 +113,20 @@ function get_room_from_jid(room_jid)
     end
 end
 
+-- Returns the room if available.
+-- @param s the current session.
+-- @return returns room if found or nil
+function get_room_from_session(s)
```

Comment:
Should I access the `session` global here instead of taking it as a parameter?

Reply:
I don't think so, as normally that is passed as an event param.